src/query.md

```diff
@@ -1,58 +1,56 @@
 # Queries: demand-driven compilation
 
+<!-- toc -->
+
 As described in [the high-level overview of the compiler][hl], the Rust compiler
 is still (as of <!-- date: 2021-07 --> July 2021) transitioning from a
-traditional "pass-based" setup to a "demand-driven" system. **The Compiler Query
-System is the key to our new demand-driven organization.** The idea is pretty
-simple. You have various queries that compute things about the input – for
-example, there is a query called `type_of(def_id)` that, given the [def-id] of
+traditional "pass-based" setup to a "demand-driven" system. The compiler query
+system is the key to rustc's demand-driven organization.
```

Comment:
I think it might still make sense to have this in bold.

Reply:
I generally don't like using bold face for emphasis in regular prose. Let me try something else here...

Reply:
Actually, I'm thinking of leaving this as-is, if that's alright. 😇

Reply:
Eh, not a big deal either way. I slightly lean toward bold in this case, but it's fine that you left it out.